logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/e2ei/usecase/EnrollE2EIUseCase.kt

```diff
@@ -55,9 +56,9 @@ class EnrollE2EIUseCaseImpl internal constructor(
 
         e2EIRepository.initE2EIClient(isNewClient = isNewClientRegistration)
 
-//         e2EIRepository.fetchTrustAnchors().onFailure {
-//             return E2EIEnrollmentResult.Failed(E2EIEnrollmentResult.E2EIStep.TrustAnchors, it).toEitherLeft()
-//         }
+        e2EIRepository.fetchTrustAnchors().onFailure {
+            //
+        }
```

Comment:
Maybe add TODO here ?

Reply:
Maybe logging? Not sure what is expected here.